.gitignore

```diff
@@ -1,8 +1,7 @@
 *.iml
 .gradle
-/local.properties
-/.idea/workspace.xml
-/.idea/libraries
+.idea
```

Comment:
Yes, I removed .idea files because Android Studio generates them when opening project, and creates them differently on different systems. And I did some sorting.